examples/udpmulticat.c

```diff
@@ -134,6 +134,8 @@ int main(int argc, char **argv)
     ubase_assert(upipe_set_uri(col_src, "@:1236"));
     ubase_assert(upipe_set_uri(row_src, "@:1238"));
 
+    usleep(200000);
```

Comment:
meuheuheuh why ?

Reply:
I was testing the cr_sys timestamping commit, but you suggested another solution so this can go